docs/htdocs/info/docs/tools/vep/script/vep_custom.html

```diff
@@ -141,7 +141,7 @@ <h2 id="custom_options">Options</h2>
   in a comma-separated list, like this: </p>
   
   <pre class="code sh_sh">
-./vep [...] --custom <b>Filename</b> , <b>Short_name</b> , <b>File_type</b> , <b>Annotation_type</b> , <b>Force_report_coordinates</b> , <b>VCF_fields</b></pre>
+./vep [...] --custom <b>file=Filename</b> , <b>short_name=Short_name</b> , <b>format=File_type</b> , <b>type=Annotation_type</b> , <b>coords=Force_report_coordinates</b> , <b>fields=VCF_fields</b></pre>
```

Comment:
I think it would be better to remove the spaces between commas in here:

```suggestion
./vep [...] --custom <b>file=Filename</b>,<b>short_name=Short_name</b>,<b>format=File_type</b>,<b>type=Annotation_type</b>,<b>coords=Force_report_coordinates</b>,<b>fields=VCF_fields</b></pre>
```